static/oas/v1/openapi.yml

```diff
@@ -20,6 +20,10 @@ paths:
     $ref: "./stats/penalties.yml#/paths/penalties"
   /user:
     $ref: "./users/users.yml#/paths/user"
+  /teams:
+    $ref: "./team/team.yml#/paths/teams"
```

Comment:
With the other comments I left you will need to change this to `"./groups/team.yml#/paths/teams"`.

Reply:
This was done.